server/mediaserver.go

```diff
@@ -635,18 +635,16 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 		go func(s *LivepeerServer, mid core.ManifestID) {
 			defer ticker.Stop()
 			for range ticker.C {
+				var lastUsed time.Time
 				s.connectionLock.RLock()
-				if cxn, exists := s.rtmpConnections[mid]; exists {
-					if time.Since(cxn.lastUsed) > refreshIntervalHttpPush {
-						s.connectionLock.RUnlock()
-						_ = removeRTMPStream(s, mid)
-						return
-					}
-				} else {
-					s.connectionLock.RUnlock()
-					return
+				if cx, exists := s.rtmpConnections[mid]; exists {
```

Comment:
We use `cxn` as a matter of convention elsewhere. Maybe preserve that? Otherwise, these changes look good.

Reply:
fixed